dataprofiler/data_readers/csv_data.py

```diff
@@ -136,6 +136,7 @@ def _guess_header_row(data_as_str, suggested_delimiter=None,
         
         # Determine type for every cell
         header_check_list = []
+        only_string_flag = True # Requires additional checks
```

Comment:
the line below split(delimiter) this needs to account for delimiters in quotes

Reply:
Done